doc/usage/quickstart.rst

```diff
@@ -320,6 +337,6 @@ More topics to be covered
        another directory, the :term:`configuration directory`.  Refer to the
        :program:`sphinx-build man page <sphinx-build>` for more information.
 
-.. |more| image:: more.png
+.. |more| image:: ../_static/more.png
```

Comment:
You can use `/_static/more.png` instead. Sphinx considers a absolute path as a relative path from srcdir.

Reply:
Done